drivers/platform/aducm3029/i2c.c

```diff
@@ -164,6 +166,8 @@ int32_t i2c_remove(struct i2c_desc *desc)
 		i2c_handler = NULL;
 	}
 	free(desc);
+	last_address = UNINITIALIZED_ADDRESS;
```

Comment:
This makes sense to be inside: if (nb_created_desc == 0)

Reply:
yeah true, fixed it, funny how i knew it's supposed to go there and placed it outside :)))